lib/stanford-mods/searchworks.rb

```diff
@@ -37,9 +37,8 @@ def sw_language_facet
                     result << SEARCHWORKS_LANGUAGES[v.strip]
                   end
                 end
-              rescue
-                # TODO:  this should be written to a logger
-                p "Couldn't find english name for #{ct.text}"
+              rescue RuntimeError
```

Comment:
I don't suppose we can rescue an even more specific exception (or better yet, not raise one at all, by doing better input validation)?